akka-cluster/src/main/resources/reference.conf

```diff
@@ -63,10 +63,9 @@ akka {
     # If this is set to "off", the leader will not move 'Joining' members to 'Up' during a network
     # split. This feature allows the leader to accept 'Joining' members to be 'WeaklyUp'
     # so they become part of the cluster even during a network split. The leader will
-    # move `Joining` members to 'WeaklyUp' after 3 rounds of 'leader-actions-interval'
-    # without convergence.
+    # move `Joining` members to 'WeaklyUp' after this configured duration without convergence.
     # The leader will move 'WeaklyUp' members to 'Up' status once convergence has been reached.
-    allow-weakly-up-members = on
+    allow-weakly-up-members = 7s
```

Comment:
In StressSpec it often became WeaklyUp with the old 3s. Here is the infolog from StressSpec that clearly shows the disadvantage of moving to WeaklyUp too early, since it requires an additional convergence round.

some WeaklyUp
```
[Title]	[Duration (ms)]	[GossipStats(gossip, merge, same, newer, older)]
join 2 to seed nodes, in 9 nodes cluster	7181	GossipStats(131,0,105,23,3)
```

no WeaklyUp
```
[Title]	[Duration (ms)]	[GossipStats(gossip, merge, same, newer, older)]
join 2 to seed nodes, in 9 nodes cluster	5202	GossipStats(81,0,68,12,1)
```